src/test/java/com/xceptance/xlt/report/providers/DummyReportProviderConfiguration.java

```diff
@@ -88,6 +91,12 @@ public int getMovingAveragePercentage()
     {
         return 0;
     }
+    
+    @Override
+    public List<MovingArerage> getAdditonalMovingAverages()
+    {
+        return Collections.emptyList();
```

Comment:
Since Java 9, List interface provides method `of` that returns type-safe immutable collections including a stable reference to empty list. Thus, there is no need for `Collections.emptyList()` anymore.

```suggestion
        return List.of();
```

This is no CR, just FYI.